de.metas.vertical.pharma/src/main/java/org/adempiere/impexp/process/ImportPharmaProduct.java

```diff
@@ -0,0 +1,14 @@
+package org.adempiere.impexp.process;
```

Comment:
Pls avoid using "adempiere" in package names. You could use "de.metas.product.impexp" so something similar.